src/modules/currencies/entities/currency.entity.ts

```diff
@@ -22,24 +23,24 @@ export class CurrencyEntity {
   isoCode: string;
 
   @Column({ name: 'DATE_CREATED_DATETIME', type: 'timestamp' })
-  @ApiProperty({ example: '2010-03-15 16:13:34.000' })
+  @ApiProperty({ example: '2017-03-06T16:20:17' })
   @Transform(({ value }) => format(value, "yyyy-MM-dd'T'HH:mm:ss"))
```

Comment:
Should we remove @Transform here and use new date format "2021-01-26T23:59:59.000Z"

Reply:
good shout - updated